src/domain/platform/admin/templates/InnovationPacks/ImportTemplatesDialogGalleryStep.tsx

```diff
@@ -46,7 +46,7 @@ const ImportTemplatesDialogGalleryStep = <T extends Template, Q extends T & Temp
           innovationPackNameID: pack.nameID,
           innovationPackId: pack.id,
         }))
-      ) as any; //TODO? Better?
+      ) as any as Q[];
```

Comment:
`T extends Template`, which has `info` common to all template types, but also has properties specific to the template type, like `value` for canvases or `defaultDescription` for Aspects.
`TemplateFromInnovationPack` has just the innovation pack information, InnovationPack ID, NameId and displayName, and the provider organization.

Q is the type of the templates coming from innovation packs. There is a new query `InnovationPacksQuery`.
`Q extends T & TemplateFromInnovationPack` this is to make type Q have all the properties of the template and also the InnovationPack information.

In these `map` I'm reading the results from the `InnovationPacks` query and turning them into `Q`. 
I am sure I have all the properties for Q because in the innovationPacks query I am using the same fragments:
```
aspectTemplates {
  ...AdminAspectTemplate
}
canvasTemplates {
  ...AdminCanvasTemplate
}
lifecycleTemplates {
  ...AdminLifecycleTemplate
}
```

With this `...template` I am passing all the properties, but I can't find a way to tell Typescript.